recovery/recovery.py

```diff
@@ -146,10 +150,9 @@ def main(scriptConf=None):
         ]
         LOGGER.info('Save completed successfully')
         job_log(start_time, temp_dir, messages=messages, status=JobLogStatuses.SUCCESS, filename=current_file_basename)
-    else:
-        file_name = ''
-        if args.name_load:
-            file_name = os.path.join(confd_backups, args.name_load)
+    elif args.load:
+        if args.file:
+            file_name = os.path.join(confd_backups, args.file)
         else:
```

Comment:
`args.file` will never be `None` since it has a default value.

Reply:
I assume that we can provide an empty value if we use a load process, as the default process is saving and we provide a default value to ```file``` for this

Reply:
It's not actually possible to start the load process. Even if the `--load` flag is passed, `args.save` is checked first, and it's default value is `True`. You can try running the script and checking `/var/yang/logs/yang.log`. The default value for `args.file` is set regardless of the `--load` and `--save` flags.

Reply:
> The default value for args.file is set regardless of the --load and --save flags.

Yeah, I understand that.

> It's not actually possible to start the load process. Even if the --load flag is passed, args.save is checked first, and it's default value is True

Maybe we should set the default values for both ```save``` and ```load``` to ```False``` then and specify the particular flag for each run of the script explicitly?

Reply:
Yes, that might be better.